src/inputs/pipe.c

```diff
@@ -364,18 +406,24 @@ parse_item(struct input_metadata *m, const char *item)
     {
       pthread_mutex_lock(&pipe_metadata_lock);
 
-      if (data != &progress)
-	free(*data);
+      if (data != &progress && data != &volume)
+        free(*data);
```

Comment:
I used 2 spaces (and extra level for braces) the original code was incorrectly indented at some places (using tab char) which was inconsistent in my editor. Did I miss something? I mean should I revert to using tab char?